src/main/java/core/basesyntax/impl/StorageItem.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.impl;
+
+public class StorageItem<K, V> {
+    private K key;
+    private V value;
```

Comment:
```suggestion
public class Node<K, V> {
    private K key;
    private V value;
```
usually classes that store values in collections are called Node
lets also make it an inner class